Update README.md (#15)
NixOS/moderation: README.md

To achieve our objectives, we are committed to maintaining transparency in our actions *while respecting the privacy of those involved*.

## Team

The moderation team is not anonymous, you can always look up its current members [on the home page](https://nixos.org/community/teams/moderation).

## Reporting issues

- E-mail: [community@nixos.org](mailto:community@nixos.org).
- Matrix room: [#moderation:nixos.org](https://matrix.to/#/#moderation:nixos.org)
- On Discourse: [moderators group](https://discourse.nixos.org/g/moderators)
- On GitHub, ping [`@NixOS/moderation`](https://nixos.org/community/teams/moderation)

Or contact any of the current team members directly, e.g. via Matrix.

The project team is obligated to maintain confidentiality with regard to the reporter of an incident.

## Practices

### Moderation decisions

The moderation team always involves multiple members in significant decisions like the suspension of a long-time contributor. We also review all decisions together. So in most cases, the entire team always stands behind and is accountable for each moderation decision.

### Moderation log

> We record supsension events into the moderation log: [moderation-log.md](moderation-log.md).

Reviewer: ```suggestion
We record suspension events into the moderation log: [moderation-log.md](moderation-log.md).
```

> In order to protect the privacy of the suspended people (and also the vicims), moderation decisions are not published. We always tell the person in question in private why they got suspended. Other trusted community members can privately ask the moderators about a specific suspension.

Reviewer 2: ```suggestion
In order to protect the privacy of the suspended people (and also the victims), moderation decisions are not published. We always tell the person in question in private why they got suspended. Other trusted community members can privately ask the moderators about a specific suspension.
```

> In order to protect the privacy of the suspended people (and also the victims), moderation decisions are not published. We always tell the person in question in private why they got suspended. Other trusted community members can privately ask the moderators about a specific suspension.

Reviewer 3: IMO the word victim can be a bit too strong (not everybody would necessarily identify as a victim when they're being attacked). This is hopefully just as clear:

```suggestion
In order to protect the privacy of the people affected by moderation decisions, these are not published. We always tell the person in question in private why they got suspended. Other trusted community members can privately ask the moderators about a specific suspension.
```

Author: Agreed. However, I'd like to explicitly point out that sometimes this is more than just about the privacy of the suspended person. How about "In order to protect the privacy of the people involved, moderation decisions are not published."?

Reviewer 4: I think that "decisions" isn't quite the right word, either (e.g. in the case of suspending a user, the decision was to suspend them, and that's recorded in the log, right?). If you agree, maybe it could be something like:

> In order to protect the privacy of the individuals involved, reasoning behind moderation decisions are not published.

Author: Good point, and sounds good to me. What about "justification" instead of "reasoning" though?

Reviewer 4: Yep, that sounds perfect to me!

Author: ```suggestion
In order to protect the privacy of the individuals involved, justification behind moderation decisions is not published. We always tell the person in question in private why they got suspended. Other trusted community members may privately ask the moderators about a specific suspension.
```
